### ISSUE TITLE
You are qualified to join but can not join

### ISSUE THREAD

**Issue by ngocnam1202** [keybase/client#20625]
When I touch on "See if you qualify" on the screen display "You are qualified to join". Then I touch on "Become a lucky airdropee", the result is "You are not qualified to join".

**Comment by ngocnam1202** [comment]
![75233539_383441639201192_7976821400541331456_n](https://user-images.githubusercontent.com/16237017/67619625-fc311980-f827-11e9-98dc-253cf4e87b42.png)
![73192897_415025312529538_3252611001350094848_n](https://user-images.githubusercontent.com/16237017/67619629-0521eb00-f828-11e9-917d-225fe535e53b.png)

**Comment by revonoc** [comment]
Same exact issue for me!

**Comment by ryanodor** [comment]
Agree. The series of checks confirm I qualify, yet clicking “Become a lucky airdropee” does not allow me to join.

**Comment by Luna1200** [comment]
Same issue here, can someone reply what's going on :/ ?

**Comment by Z86-tech** [comment]
this is a known issue https://github.com/keybase/client/issues/20264
